packages/flame/test/components/mixins/snapshot_test.dart

```diff
@@ -0,0 +1,257 @@
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/extensions.dart';
+import 'package:flame/game.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group(
+    'Snapshot',
+    () {
+      testWithGame<_SnapshotTestGame>(
+        'Snapshot should be created once',
+        _SnapshotTestGame.new,
+        (game) async {
+          final sc = game.snapshotComponent;
```

Comment:
```suggestion
          final sc = game.snapshotComponent;
```
To improve readability we don't allow abbreviations, so rename all `sc` to `snapshotComponent` for example.

Reply:
Thank you. Changed throughout.